### DISCUSSION TITLE
Firewall rules

### DISCUSSION THREAD

**Discussion by davidnetten** [jaredhendrickson13/pfsense-api#269] [Q&A] [answered]
First off, this is a really great addition to pfSense. Keep up the good work.

I'm mostly using the firewall endpoint and I noticed not everything can be changed yet. In my case:

- tcpflags on and off
- State type to sloppy
- In floating: Quick True or False

Am I correct these things are not configuring via the API?

**Comment by jaredhendrickson13** [comment, marked as answer]
Hey there!

Yes, you are correct. Those options currently cannot be set from the firewall rule endpoint. However, it's likely that these are just features that have not been requested yet. You can take a look through the open features requests on the [issues page](https://github.com/jaredhendrickson13/pfsense-api/issues) to see if anyone else has requested these features. If not, feel free to open an issue for each and I can work them into the development flow. 

Since these options are mostly just toggles, they should be relatively easy to implement. It is possible they could squeeze into [v1.5.0](https://github.com/jaredhendrickson13/pfsense-api/pull/243) which I expect to release in about a month or so.

Hope this helps.

Thanks!

**Reply by davidnetten** [reply]
Thank you for your response! I created a feature request. Again, this is really brilliant. 👍
